apps/nextjs-website/src/components/atoms/WebinarHeaderBanner/WebinarHeaderBanner.tsx

```diff
@@ -0,0 +1,63 @@
+'use client';
+import { Button, Typography, Box, useTheme, IconButton } from '@mui/material';
+import { FC, useState } from 'react';
+import VideoLibraryIcon from '@mui/icons-material/VideoLibrary';
+import CloseIcon from '@mui/icons-material/Close';
+import EastIcon from '@mui/icons-material/East';
+import { useRouter } from 'next/navigation';
+import { translations } from '@/_contents/translations';
+
+export type WebinarHeaderBannerProps = {
+  readonly slug: string;
+  readonly text: string;
+};
+
+const WebinarHeaderBanner: FC<WebinarHeaderBannerProps> = ({ slug, text }) => {
+  const [visible, setVisible] = useState(!window.localStorage.getItem(slug));
```

Comment:
I think is better to save instead of true the endDateTime of webinar

```suggestion
export type WebinarHeaderBannerProps = {
  readonly slug: string;
  readonly text: string;
  readonly endDateTime: Date;
};

const WebinarHeaderBanner: FC<WebinarHeaderBannerProps> = ({ slug, text }) => {
  const [visible, setVisible] = useState(!window.localStorage.getItem('slug') || new Date(window.localStorage.getItem('slug') || new Date().toISOString()) < new Date());
```